recipe/meta.yaml

```
@@ -48,28 +47,33 @@ requirements:
     - python
     - setuptools
     - wheel
-    - waf
 
   run:
-    - altgraph
-    - binutils  # [aarch64]
     - python
+    - setuptools >=42.0.0
+    - altgraph
+    # - binutils  # [aarch64]
     - pefile >=2022.5.30  # [win]
-    - pyinstaller-hooks-contrib >=2021.4
-    - pywin32  # [win]
-    - pywin32-ctypes  # [win]
+    - pywin32-ctypes >=0.2.1 # [win]
     - macholib >=1.8  # [osx]
-    - setuptools >=42.0.0  # due to pkg_resources import
-    - importlib-metadata >=1.4  # [py<38]
+    - pyinstaller-hooks-contrib >=2024.6
+    # - pywin32  # [win]
+    - importlib-metadata >=4.6  # [py<310]
+    - packaging >=22.0
 
 test:
   imports:
     - PyInstaller
   requires:
     - pip
-  files:
-    - hello.py
-    - multiprocessing_test.py
+    - pytest >=2.7.3
+    - execnet >=1.5.0
+    - psutil
+  source_files:
+    - tests
```

```suggestion
  source_files:
    - tests
    - setup.cfg
```
So you benefit from this configuration: 
https://github.com/pyinstaller/pyinstaller